gc-properties.adoc

```diff
@@ -0,0 +1,23 @@
+= GC properties
```

Comment:
Why is this file in the root of the project?

Can't it be moved to a specific maven project?